src/data.js

```diff
@@ -1,13 +1,48 @@
-// import data from './data/injuries/injuries.js';
-// import data from './data/lol/lol.js';
-// import data from './data/patient/patient.js';
-// import data from './data/pokemon/pokemon.js';
-// import data from './data/rickandmorty/rickandmorty.js';
-// import data from './data/steam/steam.js';
-// import data from './data/steam/worldbank.js';
-
-// esta es una función de ejemplo
-
-export const example = () => {
-  return 'example';
+
+//filtrar por generos
+
+window.rnmgo = {
+  filterData(data, condition)  {
```

Comment:
Qué significa rnmgo?

Reply:
era una abreviatura para rick and morty go